addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/handler/NestBridgeHandler.java

```diff
@@ -110,7 +97,6 @@ public void initialize() {
         logger.debug("Client Secret   {}.", config.clientSecret);
         logger.debug("Pincode         {}.", config.pincode);
```

Comment:
I think it would be better to remove the dots at the end of the log lines because it is easy to also copy these values with dots into your clipboard and start using them with your own requests while debugging. :-)